reflex/app.py

```diff
@@ -498,9 +498,9 @@ def _setup_state(self) -> None:
                     else config.cors_allowed_origins
                 ),
                 cors_credentials=True,
-                max_http_buffer_size=constants.POLLING_MAX_HTTP_BUFFER_SIZE,
-                ping_interval=constants.Ping.INTERVAL,
-                ping_timeout=constants.Ping.TIMEOUT,
+                max_http_buffer_size=environment.REFLEX_SOCKET_POLLING_MAX_HTTP_BUFFER_SIZE.get(),
```

Comment:
suggest removing `_POLLING` from the name as this also applies to the websocket max message size